### ISSUE TITLE
Add some structure to this repo :)

### ISSUE THREAD

**Issue by fl0v** [fl0v/dg#10]
Maybe group the scripts by their use case ?
But we need to investigate if this can be done without breaking old scripts update flow, now tempermonkey knows to update a script if version was incremented. An idea would be to keep copies of the scripts with old path but change the update url in meta data (i think tempermonkey knows to use it). After a while the old copies can be removed.

Need to think of a clean structure:
- Planets
- Feets
- Radar
- Navigation
basically by the page they enhance, and if a script applies to more then one just pick one dir where is best suited
